code/game/objects/items/weapons/tanks/tanks.dm

```diff
@@ -117,17 +117,18 @@
 	user.visible_message("<span class='suicide'>[user] is putting [src]'s valve to [user.p_their()] lips! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	playsound(loc, 'sound/effects/spray.ogg', 10, 1, -3)
 	if (H && !qdeleted(H))
-		for(var/obj/item/W in H)
-			H.dropItemToGround(W)
-			if(prob(50))
-				step(W, pick(alldirs))
-		H.hair_style = "Bald"
-		H.update_hair()
-		H.bleed_rate = 5
-		new /obj/effect/gibspawner/generic(H.loc, H.viruses, H.dna)
-		H.adjustBruteLoss(1000) //to make the body super-bloody
-
-	return (BRUTELOSS)
+		spawn(0)
```

Comment:
Well old code had a sleep(3) for a sort of delay I guess so that's there if anyone in the future wants to increase the delay for some reason. Like if they want the tank to kill you instantly but then gibs you like 30 seconds after or something.